src/xdocs/config_misc.xml

```diff
@@ -326,19 +326,19 @@ String unitAbbrev = "\u03bc\u03bc\u03bc";
         <p>Example #4: Comment has distributed previous statement.</p>
         <source>
 1   public void foo11() {
-2       CheckUtils
-3           .getFirstNode(new DetailAST())
-4           .getFirstChild()
-5           .getNextSibling();
+2     CheckUtils
+3       .getFirstNode(new DetailAST())
+4       .getFirstChild()
+5       .getNextSibling();
 6       // it is OK
```

Comment:
@varunu28 @romani I just noticed this file is for indentation so it has to be examined harder.
This line is incorrect. It should be in the same position as `CheckUtils`.

@romani Please examine these cases a bit harder.
I will re-review.

Reply:
@rnveach Done